### PR TITLE
CLI test tool robustness improvements

### PR DESCRIPTION
**Pull request by aqnuep**
Improvements to clitests to allow @MarkCallow to more easily investigate his issues in PR https://github.com/KhronosGroup/KTX-Software/pull/783.

@MarkCallow, please merge CTS PR https://github.com/KhronosGroup/KTX-Software-CTS/pull/6 first.

It would also be great to get write access to the CTS repo in general.

### CODE REVIEW

**Comment by MarkCallow** [comment]
You have write access to the CTS repo @aqnuep.

Once the CTS PR is merged, doesn't the ref here need to be changed?

**Comment by aqnuep** [comment]
Updated to latest main of CTS repo.

**Review by MarkCallow — APPROVED** [review]

**Pull request merged**
